examples/js/loaders/FBXLoader.js

```diff
@@ -386,6 +386,25 @@
 		texture.wrapS = valueU === 0 ? THREE.RepeatWrapping : THREE.ClampToEdgeWrapping;
 		texture.wrapT = valueV === 0 ? THREE.RepeatWrapping : THREE.ClampToEdgeWrapping;
 
+		if ( textureNode.properties.Scaling ) {
```

Comment:
The pattern used throughout the loader is 

    if ( 'Scaling' in textureNode.properties ) {

I don't really have a preference here, but it would be good to stay consistent.